### ISSUE TITLE
not able to compile

### ISSUE THREAD

**Issue by syedsami** [andromeda1913/ios_image_morpher#1]
Hi ,

I will appreciate if you help me out. I was trying to build the code but i am getting error on opencvv framework.

truncated fat file. Slice from 48459804 to 63151724 is past end of file with length 45151906 file '/imagewarp/Opencvv/opencv2.framework/opencv2' for architecture x86_64
linker command failed with exit code 1
Any help?

**Comment by andromeda1913** [comment]
Hi  yes  it seems  like  compiler  issue , try to  use   different comile
flags (BuildPhrases ) .      or change target architicture .
https://tinsuke.wordpress.com/2011/02/17/how-to-cross-compiling-libraries-for-ios-armv6armv7i386/

it's   been a while  since i used  that code   but   it was worked
perfectly    at   ios  6 -  7 on my iphone4s

2015-04-16 17:01 GMT+03:00 syedsami notifications@github.com:

**Comment by syedsami** [comment]
Yes, but now latest xcode need 64 bit architecture and i think this code will not be able to run on xcode 6.1 . The link you shared is too old ( feb 2011 ).

**Comment by troypayne** [comment]
Anyone get this compiling?
